packages/gatsby-theme-carbon/src/components/Header/Header.module.scss

```diff
@@ -1,5 +1,39 @@
 @import '~carbon-components/scss/globals/scss/layout';
 
+// Creates the blue line under the active menu item
+.isActive::after {
+  position: absolute;
+  top: 0;
+  right: 0;
+  bottom: -2px;
+  left: 0;
+  width: 100%;
+  border-bottom: 3px solid $carbon--blue-60;
+  content: '';
+}
+
+// Creates the blue line that sits to the left of an active item in a menu.
+.isActive :global(.bx--header__menu-item--current) {
+  background-color: $carbon--gray-80;
+}
+
+// Removes the blue line that sits under an active item in menu.
+.isActive :global(.bx--header__menu-item--current)::after {
+  display: none;
+}
+
+// Overrides the style from upstream Carbon to give a blue60 instead of a blue50
+.isCurrentPageOverride :global(.bx--header__menu-item--current)::after {
```

Comment:
whats the diff between is active and iscurrentpage?

Reply:
I made better descriptors of these selectors. Hopefully, they are easier to understand.